pyvista/plotting/prop3d.py

```diff
@@ -213,6 +223,19 @@ def orientation(self) -> tuple:  # numpydoc ignore=RT01
     def orientation(self, value: tuple):  # numpydoc ignore=GL08
```

Comment:
Noticed this... should we change this while in here?

```suggestion
    def orientation(self, value: Vector):  # numpydoc ignore=GL08
```

Reply:
This change to the typing has been made in 7aaaa1e0